doc/source/ray-security/index.md

```diff
@@ -0,0 +1,30 @@
+(security)=
+
+# Security 
+
+Ray is an easy-to-use framework to run arbitrary code across one or more nodes in a Ray Cluster. Ray provides fault-tolerance, optimized scheduling, task orchestration, and auto-scaling to run a given workload.
+
+To achieve performant and distributed workloads, Ray components require intra-cluster communication. This communication includes central tenants like distributed memory and node-heartbeats, as well as auxiliary functions like metrics and logs. Ray leverages gRPC for a majority of this communication.
+
+Ray offers additional services to improve the developer experience. These services include Ray Dashboard (to allow for cluster introspection and debugging), Ray Jobs (hosted alongside the Dashboard and provides the ability to submit Ray Jobs) and Ray Client (to allow for local, interactive development with a remote cluster). These services provide complete access to the Ray Cluster and the underlying compute resources.
+
+## Personas
+
+When considering the security responsibilities of running Ray, it helps to think of the different entities interacting with Ray.
+* **Developers** write code that relies on Ray. They either run a single-node Ray Cluster locally or multi-node Clusters remotely on provided compute infrastructure.
+* **Platform providers** provide the compute environment on which **Developers** run Ray.
+* **Users** interact with the output of Ray-powered applications.
+
+## Best Practices
+**Security and isolation must be enforced outside of the Ray Cluster.** Ray expects to run in a safe network environment and to act upon trusted code. Developers and platform providers must maintain these invariants to ensure the safe operation of Ray Clusters:
+
+### Deploy Ray Clusters in a governed network environment
+* Network traffic between Ray core components and Ray auxiliary components should always be in a controlled, isolated network. Access to auxiliary services should be gated with strict network controls and/or external authentication/authorization proxies.
```

Comment:
Should `core` be capitalized?

Reply:
I may reword to "between core Ray components and additional Ray components". I'm not sure if these denote "Ray Core" per-se.